applications/vanilla/controllers/api/CategoriesApiController.php

```diff
@@ -353,6 +343,70 @@ public function post(array $body) {
     }
 
     /**
+     * Prepare data for output.
+     *
+     * @param array $row
+     */
+    public function prepareRow(array &$row) {
+        if ($row['ParentCategoryID'] <= 0) {
+            $row['ParentCategoryID'] = null;
+        }
+        $row['Description'] = $row['Description'] ?: '';
+    }
+
+    /**
+     * Move a category under a new parent.
+     *
+     * @param int $id The ID of the category.
+     * @param array $body The request body.
+     * @throws NotFoundException if unable to find the category.
+     * @throws ClientException if the parent and category are the same.
+     * @throws ClientException if the parent category ID is invalid.
+     * @throws ClientException if the target parent category does not exist.
+     * @throws ClientException if trying to move a category under one of its own children.
+     * @return array
+     */
+    public function put_parentCategoryID($id, array $body) {
```

Comment:
This should be done in patch. We are affectively updating parentCategoryID of a category. 

PUT is for replacing a whole record.